From 2096ef875fd1a09380b91a225e3d8991bddabb83 Mon Sep 17 00:00:00 2001
From: Noam Camus <ncamus@habana.ai>
Date: Wed, 07 Jul 2021 12:48:19 +0300
Subject: [PATCH] [SW-0] gaudi2: double preboot size

needed since stack is too large (by 1600B)
New size match Gaudi1

Change-Id: If29819661a6e3fc663be94bad0a697b373bf79a4
---

diff --git a/arch/gaudi2/common.h b/arch/gaudi2/common.h
index b1b4374..53c1868 100644
--- a/arch/gaudi2/common.h
+++ b/arch/gaudi2/common.h
@@ -22,7 +22,7 @@
 #define HL_SPI_LEGACY_PREBOOT_OFF	0xC0000
 
 #define PPBOOT_SIZE			0x5000		/* 20KB */
-#define PREBOOT_SIZE			0x10000		/* 64KB */
+#define PREBOOT_SIZE			0x20000		/* 128KB */
 #define SRAM_SIZE			0x3000000	/* 48MB */
 #define PREBOOT_TEXT_BASE		(PPBOOT_TEXT_BASE + PPBOOT_SIZE)
 
